pkg/model/iam/iam_builder.go

```diff
@@ -766,10 +769,23 @@ func addCalicoSrcDstCheckPermissions(p *Policy) {
 	)
 }
 
+func addNodeupPermissions(p *Policy) {
+	p.unconditionalAction.Insert(
+		"ec2:DescribeInstances", // aws.go
```

Comment:
Yep. Kublet/external CCM DaemonSet needs this:
```
        "ec2:DescribeInstances",
        "ec2:DescribeRegions",
```

I just haven't put this into a dedicated function yet.

Reply:
I was a bit unclear here. With external CCM, kubelet should ask CCM for these details rather than going to cloud directly. So kubelet should not longer need these permissions.